target/linux/ath79/generic/base-files/etc/hotplug.d/firmware/11-ath10k-caldata

```diff
@@ -123,7 +123,8 @@ case "$FIRMWARE" in
 	;;
 "ath10k/pre-cal-pci-0000:00:00.0.bin")
 	case $board in
-	comfast,cf-e313ac)
+	comfast,cf-e313ac|\
+	comfast,cf-e560ac)
 		caldata_extract "art" 0x5000 0x2f20
 		ath10k_patch_mac $(mtd_get_mac_binary art 0x6)
```

Comment:
Aha, so there is an address in art 0x6. Have you checked whether there is a matching address in 0x5006?